src/components/VideoOverviewCard/VideoOverviewCard.js

```diff
@@ -0,0 +1,28 @@
+import { useData } from "../../context";
+import styles from "./VideoOverviewCard.module.css";
+import { Link } from "react-router-dom";
+
+const VideoOverviewCard = ({ videoId }) => {
+  const { videos } = useData();
+  const video = videos.find((video) => video._id === videoId);
+  return video ? (
```

Comment:
Using Ternary Operator isn't recommended here. `&&` would be better for this situation.